released/packages/coq-mathcomp-algebra-tactics/coq-mathcomp-algebra-tactics.0.2.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "sakaguchi@coins.tsukuba.ac.jp"
+
+homepage: "https://github.com/math-comp/algebra-tactics"
+dev-repo: "git+https://github.com/math-comp/algebra-tactics.git"
+bug-reports: "https://github.com/math-comp/algebra-tactics/issues"
+license: "CECILL-B"
+
+synopsis: "Ring and field tactics for Mathematical Components"
+description: """
+This library provides `ring` and `field` tactics for Mathematical Components,
+that work with any `comRingType` and `fieldType` instances, respectively.
+Their instance resolution is done through canonical structure inference.
+Therefore, they work with abstract rings and do not require `Add Ring` and
+`Add Field` commands. Another key feature of this library is that they
+automatically push down ring morphisms and additive functions to leaves of
+ring/field expressions before normalization to the Horner form."""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.13" & < "8.15~")}
```

Comment:
```suggestion
  "coq" {(>= "8.13" & < "8.16~")}
```
The package looks good, but maybe you want to allow Coq 8.15+rc1 as well @pi8027? If it turns out there is some incompatibility, we will find out through the opam bench later and can make a change to disallow 8.15.

Reply:
IIUC, once we have a release of Coq-Elpi compatible with Coq 8.15, we could widen this constraint. cc: @gares

Reply:
please relax the bound